utils/get_abstracts.py

```diff
@@ -0,0 +1,84 @@
+import synapseclient
+from synapseclient import Table
+import argparse
+import pandas as pd
+import requests
+import json
+
+
+### Login to Synapse ###
+def login():
+
+    syn = synapseclient.Synapse()
+    syn.login()
+
+    return syn
+
+
+def get_args():
+
+    parser = argparse.ArgumentParser(
+        description='Get synapse publications table id')
+    parser.add_argument('table_id',
+                        type=str,
+                        help='Synapse table id to upload manifest to.')
+
+    return parser.parse_args()
+
+
+def get_df(syn, publications_table_id):
+
+    pubs_query = (f"SELECT pubMedId, abstract FROM {publications_table_id}")
+    pubs_df = syn.tableQuery(pubs_query).asDataFrame().fillna("")
+
+    pmid_list = pubs_df['pubMedId'].tolist()
+
+    return pubs_df
+
+
+def get_pmids(pubs_df):
+
+    # Convert column of pubmedIds to a list
+    pmid_list = pubs_df['pubMedId'].tolist()
+
+    return (pmid_list)
+
+
+def get_abstracts(pmid_list, pubs_df):
+
+    for pmid in pmid_list:
+
+        endpoint = f"https://www.ncbi.nlm.nih.gov/research/pubtator-api/publications/export/pubtator?pmids={pmid}&concepts=none"
+        response = requests.get(endpoint)
+        response_string = response.text
+        # Parse text to extract only the abstract
+        index = response_string.find('|a|') + 3
```

Comment:
This works!  You can also utilize `split()` in lieu of `find()` + string parsing:

```
abstract = response_string.split("|")[4]
```

Reply:
I wasn't sure if the abstract would always be in the fifth column because it is a text file? I just noticed that it is always after the publication title and the `|a|`. I'll just keep it as is!

Reply:
Sounds good!! 🎉 

> always be in the fifth column because it is a text file?

^ One quick clarification regarding this, as I don't want you to misunderstand that I'm saying the abstract is at the 4th index of the string.  Rather: when you use `split()`, the result is a list of the words from the string, separated by the delimiter you pass it.  So in this case, `response_string` is being split by `|`, where the resulting list is something like this:

index | value
--: | :--
0 | 28483577
1 | t
2 | Formoterol and fluticasone propionate combination improve...
3 | a
4 | BACKGROUND: The addition of long-acting beta2-agonists (LABAs) to corticosteroids...

And it's this list where I'm saying the abstract is in the 4th index (aka 5th column)